src/uu/factor/Cargo.toml

```diff
@@ -12,11 +12,11 @@ categories = ["command-line-utilities"]
 edition = "2018"
 
 [build-dependencies]
-num-traits = "0.2" # used in src/numerics.rs, which is included by build.rs
+num-traits = "0.2.13" # used in src/numerics.rs, which is included by build.rs
 
 
 [dependencies]
-num-traits = "0.2"
+num-traits = "0.2.13"
```

Comment:
because a missing feature has been added in 0.2.13?

Reply:
Yes, `OverflowingAdd` was added in 0.2.13